packages/frontend-web/src/app/scenes/Comments/components/NewComments/NewComments.tsx

```diff
@@ -1017,7 +1017,7 @@ export class NewComments extends React.Component<INewCommentsProps, INewComments
   @autobind
   onTagButtonClick(tagId: string) {
     const ids = this.getSelectedIDs();
-    this.triggerActionToast('tag', ids.length, () => this.props.tagComments(ids, tagId));
+    this.triggerActionToast('Accept', ids.length, () => this.props.tagComments(ids, tagId));
```

Comment:
I think this is just wrong.  I'm going to have to reassess...

Reply:
Fixed this.  It was an incorrect change.  Not sure how it happened.

Reply:
cool, we may want to think about a test for this then?

Reply:
We don't yet write tests for any of these large components with significant internal state.  I'm not sure storyboard is up to the task.  It would be nice to have some though.  I'll think about it, and do some research.